cs5530/UserService.java

```diff
@@ -0,0 +1,50 @@
+package cs5530;
+
+import java.lang.*;
+import java.sql.*;
+import java.io.*;
+//import javax.servlet.http.*;
+
+public class UserService {
+    // userType 0 is UberUser
+    // userType 1 is UberDriver
+    public UserService() { }
+
+    public boolean createUberUser(Statement stmt, String login, String password, String name, String address, String phone) {
+        this.createUser(stmt, login, password, name, address, phone, 0);
+    }
+
+    public boolean createUberDriver(Statement stmt, String login, String password, String name, String address, String phone) {
+        this.createUser(stmt, login, password, name, address, phone, 1);
```

Comment:
You shouldn't need to use "this" in Java. You should be able to straight up call "createUser"